### DISCUSSION TITLE
Add support for Video-LLaVA

### DISCUSSION THREAD

**Discussion by psychodinae** [ggml-org/llama.cpp#5661] [Ideas]
https://github.com/PKU-YuanGroup/Video-LLaVA

https://huggingface.co/spaces/LanguageBind/Video-LLaVA

What sort of black magic is this? Is it possible to implement as in LLaVA 1.5?

An GGUF version of this would be amazing.

**Comment by ThePBJain** [comment]
+1 I tried to use the LLaVA image encoder converter in the examples and it doesn't seem to work since Video-LLaVA doesn't generate the `llava.clip` file after you run `llava-surgery-v2.py`.

I looked and found this line:

```
Searching for vision tower tensors in .\Video-LLaVA-7B\model-00002-of-00002.safetensors
No vision tower found in .\Video-LLaVA-7B\model-00002-of-00002.safetensors
...
Searching for vision tower tensors in .\Video-LLaVA-7B\model-00001-of-00002.safetensors
No vision tower found in .\Video-LLaVA-7B\model-00001-of-00002.safetensors
Done! All vision tower tensors are removed from the model files and stored in llava.clip file.
```

Not sure if its failing to see the video_tower and image_tower tensors...
